app/controllers/api/functions.php

```diff
@@ -1433,7 +1433,8 @@
         $response->noContent();
     });
 
-App::post('/v1/functions/:functionId/deployments/:deploymentId/builds/:buildId')
+App::post('/v1/functions/:functionId/deployments/:deploymentId/build')
+    ->alias('/v1/functions/:functionId/deployments/:deploymentId/builds/:buildId')
     ->groups(['api', 'functions'])
     ->desc('Create build')
```

Comment:
```suggestion
    ->desc('Rebuild deployment')
```

Reply:
@Meldiron what should be the sdk method called?